radicle-cli/src/commands/patch.rs

```diff
@@ -158,13 +159,13 @@ impl Args for Options {
                     unknown => anyhow::bail!("unknown operation '{}'", unknown),
                 },
                 Value(val) if op == Some(OperationName::Show) && patch_id.is_none() => {
-                    patch_id = Some(term::cob::parse_patch_id(val)?);
+                    patch_id = Some(val.to_string_lossy().to_string());
```

Comment:
It might be nice to clean this up by adding a new function in here https://github.com/radicle-dev/heartwood/blob/master/radicle-cli/src/terminal/args.rs:
```rust
pub fn string(val: &OsString) -> String {
    val.to_string_lossy().to_string()
}
```
then we could do:
```suggestion
                    patch_id = Some(args::string(val));
```